src/game/SpellEffects.cpp

```diff
@@ -1151,7 +1151,7 @@ void Spell::EffectDummy(SpellEffectIndex eff_idx)
                 }
                 case 32027:                                 // Expedition Flare
                 {
-                    if (!m_CastItem)                        // 32029 = Expedition Preserver | 32030 = Expedition Scout
+                        // 32029 = Expedition Preserver | 32030 = Expedition Scout
                         m_caster->CastSpell(m_caster, (urand(0, 1) ? 32029 : 32030), true, m_CastItem);
```

Comment:
Well I was criticizing that the spells wont work when cast alone, i.e. the CastItem checks at the start of dummies. Once again to get you a definitive answer I would have to launch it and confirm it myself. Last few items i dealt with, always had m_CastItem just fine in dummy effect, but then again, I am not saying items arent getting corrupted somewhere ^^